data/pigui/modules/new-game-window/flight-log.lua

```diff
@@ -233,6 +233,44 @@ FlightLog.reader = Helpers.versioned {{
 
 		return value
 	end
+
+}, {
+
+	version = 90,
```

Comment:
Will this hardcoding of the version need to be updated or changed at some point? Or it's only there while we want to have compatibility with versions prior to 90? 
(Sorry, I'm too lazy to read the code)

Reply:
This number is here so that the following function is called for saves v90 and newer. Immediately before you can see the function for v89, it will be called when restoring saves v89 and older. This list can grow on both sides to support older and newer save versions.